pipeline.py

```diff
@@ -92,8 +93,11 @@ def validate_dataset(path_to_validate):
 
 
 def main():
```

Comment:
all starting code should be written under the If __main__ == "__name__": line

Reply:
I've understood, will fix it